### ISSUE TITLE
Update peerpad.net

### ISSUE THREAD

**Issue by parkan** [ipfs-shipyard/peer-pad#288]
We've done a fair amount of work on peerpad master, and dev.peerpad.net has served us (mostly) well. We also haven't seen much evidence that the explanatory landing page etc on the main site bring a lot of value, given our developer focus.

Let's update peerpad.net to the current peer-pad version so visitors are met with a more stable, better-performing version.

This may be blocked until #287 is in, depending on Jenkins health

**Comment by parkan** [comment]
(once this is done, we can also periodically update prod -- probably a good topic for the bi-weekly meeting)

**Comment by pgte** [comment]
@parkan Yes, I think that's the best approach for persistence: to delegate to a pinner cluster.
There's still one problem: where can the pinner store the HEAD of the collaboration in non-ephemeral memory? IPNS would be a good candidate, once we get it usable on js-ipfs...

**Comment by parkan** [comment]
looks like 0.34.x has IPNS: https://github.com/ipfs/js-ipfs/releases/tag/v0.34.0
